File: uikit/src/main/java/com/sendbird/uikit/activities/viewholder/EmojiReactionViewHolder.java
package com.sendbird.uikit.activities.viewholder;

import androidx.annotation.NonNull;

import com.sendbird.android.message.Reaction;
import com.sendbird.uikit.databinding.SbViewEmojiReactionBinding;

public class EmojiReactionViewHolder extends BaseViewHolder<Reaction> {
    @NonNull
    private final SbViewEmojiReactionBinding binding;

    public EmojiReactionViewHolder(@NonNull SbViewEmojiReactionBinding binding) {
        super(binding.getRoot());
        this.binding = binding;
    }

    @Override
    public void bind(@NonNull Reaction item) {
        this.binding.emojiReactionView.drawReaction(item);
    }
}

File: uikit/src/main/java/com/sendbird/uikit/internal/ui/viewholders/EmojiReactionViewHolder.kt
package com.sendbird.uikit.internal.ui.viewholders

import com.sendbird.android.message.Reaction
import com.sendbird.uikit.activities.viewholder.BaseViewHolder
import com.sendbird.uikit.databinding.SbViewEmojiReactionBinding

internal class EmojiReactionViewHolder(
    private val binding: SbViewEmojiReactionBinding
) : BaseViewHolder<Reaction>(binding.root) {
    override fun bind(item: Reaction) {
        binding.emojiReactionView.drawReaction(item)
    }
}
